manifests/testing/iscsi-demo-target.yaml.in

```diff
@@ -1,3 +1,28 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubevirt-iscsi
```

Comment:
I would prefer to not create for every new testing infrastructure an extra service account. Could you change that to `kubevirt-testing` and the ClusterRole to `kubevirt-testing-cluster-admin`? It would then be obvious that we can use it for every testing component.

Reply:
Sure.
Becuase this account will be used for every testing object, should it be declared in a separate manifest?

Reply:
Sounds good. :+1: